.github/workflows/labeler-review.yml

```
@@ -0,0 +1,23 @@
+name: "Labels: Approved"
+on:
+  pull_request_review:
+    types: [submitted]
+jobs:
+  add_label:
+    # Change the repository name after you've made sure the team name is correct for your fork!
+    if: ${{ (github.repository == 'space-wizards/space-station-14') && (github.event.review.state == 'APPROVED') }}
```

Added a team for this, if we change the repo, think this should JustWork™ if that LABELER_PAT is set

```suggestion
    # Frontier: repo changed (space-wizards/space-station-14 < new-frontiers-14/frontier-station-14)
    if: ${{ (github.repository == 'new-frontiers-14/frontier-station-14') && (github.event.review.state == 'APPROVED') }}
```